src/pdb/mod.rs

```diff
@@ -664,6 +667,30 @@ pub struct PlaylistEntry {
     /// ID of the playlist.
     playlist_id: PlaylistTreeNodeId,
 }
+
+/// Contains the kinds of Metadata Categories tracks can be browsed by
+/// on CDJs.
+#[binrw]
+#[derive(Debug, PartialEq, Eq, Clone)]
+#[brw(little)]
+pub struct ColumnEntry {
+    // possibly the primary key, though I don't know if that would
+    // make sense as I don't think there are references to these
+    // rows anywhere else. This could be a stable ID to identify
+    // a category by in hardware (instead of by name).
+    id: u16,
+    // maybe a bitfield containing infos on sort order and which
+    // columns are displayed.
+    unknown0: u16,
+    /// TODO Contained string is prefixed by the "interlinear annotation"
```

Comment:
What exactly is TODO here?

Reply:
Well we need to discuss how we handle these superfluous unicode characters.

Reply:
I'd say we either keep it like you implemented, or make these separate fields before and after the string, for example`#[br(temp, assert(column_name_before = 0xFFFA)]`.

But the current comment explains how the column name is encoded, but not what's left "to do" here.

```suggestion
    /// Contained string is prefixed by the "interlinear annotation"
```

Reply:
Well you can't really make it a separate field that contains these characters the way DeviceSQLStrings are implemented. I'd rather go with a `#[bw(try_map = ...)]` that maps from a simple `String` (or the higher-level enum) to the DeviceSQLString with the prefixes (and does asserting and stripping when doing the reading). 

IMO with the current implementation, the API is not self-explanatory (since you need to manually add the prefixes when constructing the string), so thats why I think a TODO is appropriate here so we remember to improve that. I'd be happy to reword the TODO if you think we should make this clearer.